packages/esm-patient-forms-app/src/hooks/use-forms.ts

```diff
@@ -67,9 +67,12 @@ export function useForms(patientUuid: string, startDate?: Date, endDate?: Date,
   let formsToDisplay = cachedOfflineFormsOnly
     ? data?.filter((formInfo) => isValidOfflineFormEncounter(formInfo.form, htmlFormEntryForms))
     : data;
-  formsToDisplay = formsToDisplay?.filter((formInfo) =>
-    userHasAccess(formInfo?.form?.encounterType?.editPrivilege?.display, session?.user),
-  );
+
+  if (session?.user) {
+    formsToDisplay = formsToDisplay?.filter((formInfo) =>
+      userHasAccess(formInfo?.form?.encounterType?.editPrivilege?.display, session?.user),
```

Comment:
One very small simplification:

```ts
userHasAccess(formInfo?.form?.encounterType?.editPrivilege?.display, session.user)
```

Reply:
Well @ibacher, the comma is auto-appended on save

Reply:
The point was removing the last `?`, not the comma...